### ISSUE TITLE
Prometheus Config Options Required

### ISSUE THREAD

**Issue by wkelton** [cooperspencer/gickup#82]
I updated my docker container had hit this error:
```
gickup    | 2022-01-02 19:39:57 INF Starting Prometheus listener endpoint= listenAddr=
gickup    | panic: http: invalid pattern
gickup    |
gickup    | goroutine 1 [running]:
gickup    | net/http.(*ServeMux).Handle(0x112aae0, {0x0, 0x0}, {0xc7b220, 0xc000033680})
gickup    |     /usr/local/go/src/net/http/server.go:2435 +0x24c
gickup    | net/http.Handle(...)
gickup    |     /usr/local/go/src/net/http/server.go:2484
gickup    | gickup/metrics/prometheus.Serve({{0x0, 0xc0001b7428}, {0x0, 0x8}})
gickup    |     /go/src/github.com/cooperspencer/gickup/metrics/prometheus/prometheus.go:56 +0xc5
gickup    | main.main()
gickup    |     /go/src/github.com/cooperspencer/gickup/main.go:232 +0xddb
```

It appears that it's required to configure prometheus options. After putting the metrics options with the defaults listed in the example config, the container runs fine. But, there doesn't not appear a way to disable the metrics.

**Comment by cooperspencer** [comment]
Hi,
Sorry it is already quite late in my timezone. I will fix it tomorrow.

**Comment by cooperspencer** [comment]
Should be fixed with d52d7cc
You can download the new release after the pipelines are done.

**Comment by wkelton** [comment]
Confirmed fixed. Thanks!
